shared/models/conversion.py

```diff
@@ -20,6 +21,8 @@
     PresentationExchange,
 )
 
+logger = get_logger(__name__)
```

Comment:
Ah, because the `__name__` for this module is in shared, it will log to what appears to be a new service, called shared. Same if file logging was activated, it would write to a directory called shared. 

So, the recommendation is to pass the logger from wherever these functions are called. i.e.  
```py
from logging import Logger
...
def v1_presentation_state_to_rfc_state(state: Optional[str], logger: Logger) -> Optional[str]:
```

And then add Logger to: `presentation_record_to_model(.., logger: Logger)`

Then, wherever that is called in webhooks, or the verifier facades, pass the relevant logger from there